README.md

```diff
@@ -97,80 +91,98 @@ If you don’t want to send data directly as it enters S3, you can also use SNS/
+| CloudWatchLogGroupName | Provide a comma-separated list of CloudWatch log group names to monitor, for example, (`log-group1`, `log-group2`, `log-group3`). |               | :heavy_check_mark: |
 
 ### SNS Configuration
 
 To receive SNS messages directly to Coralogix, use the `SNSIntegrationTopicARN` parameter. This differs from the above use of `SNSTopicArn`, which notifies based on S3 events.
 
-| Parameter | Description | Default Value | Required |
-|---|---|---|---|
-| SNSIntegrationTopicARN | Provide the ARN of the SNS topic to which you want to subscribe for retrieving messages. |   | :heavy_check_mark: | 
+| Parameter              | Description                                                                              | Default Value | Required           |
+|------------------------|------------------------------------------------------------------------------------------|---------------|--------------------|
+| SNSIntegrationTopicARN | Provide the ARN of the SNS topic to which you want to subscribe for retrieving messages. |               | :heavy_check_mark: |
 
 ### SQS Configuration
 
 To receive SQS messages directly to Coralogix, use the `SQSIntegrationTopicARN` parameter. This differs from the above use of `SQSTopicArn`, which notifies based on S3 events.
 
-| Parameter | Description | Default Value | Required |
-|---|---|---|---|
-| SQSIntegrationTopicARN | Provide the ARN of the SQS queue to which you want to subscribe for retrieving messages. |   | :heavy_check_mark: |
+| Parameter              | Description                                                                              | Default Value | Required           |
+|------------------------|------------------------------------------------------------------------------------------|---------------|--------------------|
+| SQSIntegrationTopicARN | Provide the ARN of the SQS queue to which you want to subscribe for retrieving messages. |               | :heavy_check_mark: |
 
 ### Kinesis Configuration
 
 We can receive direct [Kinesis](https://aws.amazon.com/kinesis/) stream data from your AWS account to Coralogix. Your Kinesis stream ARN is a required parameter in this case.
 
-| Parameter | Description | Default Value | Required |
-|---|---|---|---|
-| KinesisStreamARN | Provide the ARN of the Kinesis Stream to which you want to subscribe for retrieving messages. |   | :heavy_check_mark: |
+| Parameter        | Description                                                                                   | Default Value | Required           |
+|------------------|-----------------------------------------------------------------------------------------------|---------------|--------------------|
+| KinesisStreamARN | Provide the ARN of the Kinesis Stream to which you want to subscribe for retrieving messages. |               | :heavy_check_mark: |
 
-### Generic Configuration (Optional)
+### Kafka Configuration
```

Comment:
Maybe we should combine MSK/Kafka and just write that in MSK only 2 params are needed?